Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -460,6 +460,32 @@ public void Wzhzhzh(EntityUid uid, MicrowaveComponent component, EntityUid? user
             UpdateUserInterfaceState(uid, component);
         }
 
+        public void StopCooking(Entity<ActiveMicrowaveComponent?, MicrowaveComponent> ent, bool failure)
+        {
+            if (!Resolve(ent, ref ent.Comp1))
```

Comment:
Tests are currently failing due to this line, as `ActiveMicrowaveComponent` can be null here. It'd probably be better to not pass `ActiveMicrowaveComponent` as an arg, but rather, opt for a `TryComp<ActiveMicrowaveComponent>(etc etc etc)`

Reply:
Should be fixed. Ran the integration tests locally successfully under Rider 2023.3.